Sources/SwiftGenKit/Parsers/Filter.swift

```diff
@@ -33,6 +33,17 @@ public struct Filter {
   }
 }
 
+extension Parser {
+  /// Create the regex for a filter for the given list of file extensions. The extensions will be matched at the end
+  /// of a filename, and will be matched case-insensitively.
+  ///
+  /// - Parameter extensions: The list of file extensions. Note that these can contain regex syntax if needed.
+  /// - Returns: The regex as a `String`.
+  static func filterRegex(for extensions: [String]) -> String {
+    "[^/]\\.(?i:\(extensions.joined(separator: "|")))$"
```

Comment:
We should `NSRegularExpression.escape(…)` those extensions. I mean, in practice it's quite unlikely that we'd ever have an extension that uses anything other than alphanumeric characters, but still, for correctness' sake… :wink: